src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -591,6 +601,18 @@ public int getCitizenCount()
         return citizenCount;
     }
 
+    /**
+     * Returns the maximum amount of citizen in the colony considering guard towers
+     *
+     * @return maximum amount of citizens.
+     */
+    @Override
+    public int getLimitCitizenCount()
+    {
+        return citizenCountWithEmptyGuardTowers;
```

Comment:
Better documentation, had to look at why this is relevant.
Also how is this calculated. If it is calculated directly this needs tobe changed: Each Building on its own needs to consider if it counts towards the citizen limit.
Reflect this in the documentation.

Reply:
I added some doc to this, I won't rework how the entire calculating for this is handled. That would be part of a separate PR.

Reply:
That is fine, as long as the documentation is up to date.